src/golang/lib/models/shared/level.go

```diff
@@ -0,0 +1,11 @@
+package shared
+
+type Level string
```

Comment:
nit: I feel like `Level`, `Status` and `notificationAssociation` can all go in a single file called `notification.go` since they are notification specific fields